src/actors/apache_scanner/apache_scanner.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python2
+
+"""
+Author: tmeszaro@redhat.com
+
+Actor that extracts selected data from the Augeas actor output.
+If you want to extract additional data from the augeas aug_httpd lens, check
+'searches' tuple in the printall() function.
+
+
+Possible improvements:
+- include information from: httpd -V
+- include information from: httpd -S
+- possibly identify if some testing configuration files have nested/recursive
+   Includes and write code to extract those files as well
+"""
+
+import sys
+import json
+
+
+def searchblock(tablelist, mykey, myvalue=None, block=True):
```

Comment:
Please fix function names, follow the rules from https://www.python.org/dev/peps/pep-0008/#function-names.
The same with variables. I would also like to replace names ```mykey``` and ```myvalue``` to something different, those names don't give me idea what is this.

 ```tablelist``` in my personal opinion I wouldn't join name and type, especially in Python, look you are checking if tablelist is a dict, so in that particular moment there is chance that tablelist is not a list, it's confusing.